.kokoro/datacatalog/common.cfg

```diff
@@ -0,0 +1,22 @@
+# Format: //devtools/kokoro/config/proto/build.proto
```

Comment:
This file is not necessary if you move `.kokoro/datacatalog/datacatatlog.cfg` to `.kokoro/datacatalog.cfg`. I doubt that we will need more than one build config for datacatalog samples, so that is my recommendation.

Reply:
See examples such as https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/.kokoro/endpoints-getting-started-grpc.cfg

Reply:
I see, do I need to add the:
`env_vars: {
    key: "TRAMPOLINE_BUILD_FILE"
    value: "github/nodejs-docs-samples/.kokoro/build.sh"
}`
To datacatalog.cfg?

Reply:
Followed the given example and added that env var.